src/dmd/mars.d

```diff
@@ -902,21 +902,8 @@ version (NoMain) {} else
      */
     extern (C) int _Dmain(char[][])
     {
-        import core.memory;
         import core.runtime;
 
-        // Older host druntime versions need druntime to be initialized before
-        // disabling the GC, so we cannot disable it in C main above.
-        static if (isGCAvailable)
-        {
-            if (!mem.isGCEnabled)
-                GC.disable();
-        }
-        else
-        {
-            GC.disable();
```

Comment:
This is skipped now, in the unlikely event of `!isGCAvailable`.

Reply:
Good catch! I restored that part.

Reply:
I think you'll still need the `core.memory` import.

Reply:
> I think you'll still need the `core.memory` import.

Oops, restored. I also now moved setting rt_options to mars.d, where you would usually deal with these options to begin with.